src/trainer/ResultsView.js

```diff
@@ -18,86 +17,80 @@ const plotUrl = (jobWithDate, plotType) => {
   );
 };
 
-const ResultsView = ({ selectedPlot, parentStyles }) => {
+const ResultsView = ({ selectedPlot }) => {
   return (
-    <Grid container spacing={8} style={parentStyles.grid}>
-      <Grid item xs={12}>
-        <h2>Results from {selectedPlot}</h2>
-      </Grid>
-      <Grid item xs={12}>
-        <Paper>
+    <List>
+      <h2>Results from {selectedPlot}</h2>
+      <List.Item>
+        <Card title="Loss History">
           <img
             src={plotUrl(selectedPlot, 'loss')}
```

Comment:
Quality-wise, I think it would be nicer if we created plots with javascript (e.g. chart.js) instead of uploading matplotlib images. This would also make it easier to compare graphs (since it's just moving around arrays within the charting library). See [this](http://equisat.brownspace.org/data/historical/advanced) for an example.

Same deal with true negatives / false negatives / etc. If we stored a list of true negatives / false negatives / etc. on MongoDB, we would be able to bring up the entire list of false negatives for users to see with little additional storage (since the images are already saved in preprocessing), which is more useful compared to showing 25 as it is right now.

Reply:
chart.js would be great. Integrating the database makes a lot of sense

#29 and #30